pkg/minikube/cluster/commands.go

```diff
@@ -32,6 +32,8 @@ var startCommandFmtStr = `
 PATH=/usr/local/sbin:$PATH nohup sudo /usr/local/bin/localkube %s --generate-certs=false --logtostderr=true > %s 2> %s < /dev/null &
 `
 
+var localkubeDownloadFmtStr = `sudo curl -sSL --retry 5 -o /usr/local/bin/localkube %s && sudo chmod 0777 /usr/local/bin/localkube`
```

Comment:
If using curl from the vm, proxy env vars http_ host, etc will need setting in the vm if behind a proxy. Wonder if it's better to download on host via go HTTP client, respecting env vars, & streaming straight to vm filesystem over SSH tunnel?